CONTRIBUTING.md

```diff
@@ -156,7 +156,6 @@ You should have [ESLint running in your editor](http://eslint.org/docs/user-guid
 
 > Please do not ignore any linting errors, as they are meant to **help** you and to ensure a clean and simple code base.
 
-Make sure none of your JavaScript is longer than 80 characters per line. The reason we enforce this is because one of our dependent NPM modules, [jsonlint](https://github.com/zaach/jsonlint), does not fully support wildcard paths in Windows.
```

Comment:
@Bouncey So what's the final verdict on this? 
My reasons for removing it: 
- This rule seemed to have nothing to do with jsonlint.
- There are a bunch of rules, but this is the only one that is mentioned. That could be misleading (it might be seen as the only important one).